src/stdlib/math.md

```diff
@@ -214,6 +214,11 @@ The type `T` below substitutes either `f32` or `f64` depending on the implementa
   ```
   Returns the value of `x` rounded to the nearest integer.
 
+* ```ts
+  function seedRandom(value: i64): void
+  ```
+  Seeds the `random` function. Called internally by `random` if the seed hasn't been set.
```

Comment:
I don't know if this the best way to say this so any thoughts on this would be nice.

Reply:
Also not sure if this should be mentiond in the documentation for `random` or not.